Makefile

```diff
@@ -26,6 +26,10 @@ test: test-unit
 test-unit:
 	go test $(TESTS) -run 'Unit' -coverprofile=coverage.out
 
+.PHONY: test-integration
+test-integration:
+	go test $(TESTS) -run 'Integration' -coverprofile=coverage.out
```

Comment:
This doesn't correspond with the styleguides

Reply:
`coverprofile` flag removed 👍